department-of-veterans-affairs/caseflow
# Indicate that HP should be used with Chrome

@sharonwarner and @mdbenjam  I also wanted to use the same logic for Reader since they have the same ticket https://github.com/department-of-veterans-affairs/caseflow/issues/4133. 

Resolves #4968

### Description
This PR creates a new page that shows that Hearing Prep can not be opened in Internet Explorer

### Acceptance Criteria 
- [ ] 1. When Hearing Prep is opened in a browser other than Chrome display the following message:

**[Browser Name] is not supported**
To access Hearing Prep, you must use Chrome as your browser.
If you need to install Chrome on your computer, please call the
VA Enterprise Service Desk at 855-673-4357.

### Testing Plan
1. In order to check that page opens in IE. Go to Safari
2. Click on Develop.
3. Click on UserAgent and change it to  Internet Explorer.
4. Refresh page and see that New page preventing user from using IE.

![hearing](https://user-images.githubusercontent.com/23080951/38902760-e996f6a6-426f-11e8-81fa-400de7436f18.gif)

5. Go to chrome. Refresh page and see Hearing-prep
**Safari**
<img width="1170" alt="screen shot 2018-04-20 at 1 52 52 pm" src="https://user-images.githubusercontent.com/23080951/39066229-576b9590-44a2-11e8-9974-dcb7bc99c26a.png">

**Firefox**
<img width="1228" alt="screen shot 2018-04-20 at 11 27 43 pm" src="https://user-images.githubusercontent.com/23080951/39080274-82b51a7c-44f8-11e8-9396-1ce6153d9fe0.png">
